src/components/FButton.vue

```diff
@@ -446,7 +445,7 @@ const props = withDefaults(defineProps<FButtonProps>(), {
 });
 
 const emit = defineEmits<{
-  (name: 'click', e: MouseEvent | KeyboardEvent): void;
+  (name: 'click', e: PointerEvent): void;
```

Comment:
I'm surprised, so :
- a click emits a `PointerEvent` ? always ? cross-browser ? (I did not see any documentation on it)
- `PointerEvent` and `MouseEvent` interfaces are not the same, how vue-router’s `navigate` does not complain ?